hotshot-state-prover/src/circuit.rs

```diff
@@ -292,32 +283,32 @@ where
         .iter()
         .flat_map(|var| [var.state_ver_key.0.get_x(), var.state_ver_key.0.get_y()])
         .collect::<Vec<_>>();
-    let state_ver_key_comm = RescueNativeGadget::<F>::rescue_sponge_with_padding(
+    let _state_ver_key_comm = RescueNativeGadget::<F>::rescue_sponge_with_padding(
         &mut circuit,
         &state_ver_key_preimage_vars,
         1,
     )?[0];
-    circuit.enforce_equal(
-        state_ver_key_comm,
-        lightclient_state_pub_var.stake_table_comm().state_keys_comm,
-    )?;
+    // circuit.enforce_equal(
+    //     state_ver_key_comm,
+    //     lightclient_state_pub_var.stake_table_comm().state_keys_comm,
+    // )?;
 
     // checking the commitment for the list of stake amounts
     let stake_amount_preimage_vars = stake_table_var
         .iter()
         .map(|var| var.stake_amount)
         .collect::<Vec<_>>();
-    let stake_amount_comm = RescueNativeGadget::<F>::rescue_sponge_with_padding(
+    let _stake_amount_comm = RescueNativeGadget::<F>::rescue_sponge_with_padding(
         &mut circuit,
         &stake_amount_preimage_vars,
         1,
     )?[0];
-    circuit.enforce_equal(
-        stake_amount_comm,
-        lightclient_state_pub_var
-            .stake_table_comm()
-            .stake_amount_comm,
-    )?;
```

Comment:
realized that these are wrong, we need these to ensure security. 

sry that I miss it during my last review.

Reply:
fixed in [ab81b0b](https://github.com/EspressoSystems/espresso-sequencer/pull/1986/commits/ab81b0bad7497dbcedc4434cd1881cb6891d7c58)